modules/dev-utils/src/main/java/org/apache/ignite/development/utils/IgniteWalConverter.java

```diff
@@ -34,10 +37,28 @@
 import org.apache.ignite.logger.NullLogger;
 import org.jetbrains.annotations.Nullable;
 
+import static java.util.stream.Collectors.toList;
+import static org.apache.ignite.IgniteSystemProperties.IGNITE_TO_STRING_INCLUDE_SENSITIVE;
+import static org.apache.ignite.IgniteSystemProperties.getBoolean;
+import static org.apache.ignite.IgniteSystemProperties.getEnum;
+import static org.apache.ignite.development.utils.ProcessSensitiveData.HIDE;
+import static org.apache.ignite.development.utils.ProcessSensitiveData.SHOW;
+
 /**
  * Print WAL log data in human-readable form.
  */
 public class IgniteWalConverter {
+    /**
+     * System property for printing {@link WALRecord}. By default, {@code false}.
+     */
+    static final String PRINT_RECORDS = "PRINT_RECORDS";
+
+    /**
+     * System property for setting {@link ProcessSensitiveData strategy} of output sensitive data.
+     * By default, {@link ProcessSensitiveData#SHOW}.
+     */
+    static final String SENSITIVE_DATA = "SENSITIVE_DATA";
```

Comment:
Why do we need another system property for sensitive data? We can just rework IGNITE_TO_STRING_INCLUDE_SENSITIVE by adding new possible values.

Reply:
We decided with @akalash that we would do something different and not change the current logic. This property is only used here.